habitat_baselines/rl/ppo/ppo.py

```diff
@@ -213,10 +218,11 @@ def after_backward(self, loss: Tensor) -> None:
         pass
 
     def before_step(self) -> None:
-        if False:
-            nn.utils.clip_grad_norm_(
-                self.actor_critic.parameters(), self.max_grad_norm
-            )
+        pass
```

Comment:
Beware this looks like something I disabled while iterating on runtime perf. We should probably re-enable this code.

Reply:
Yes, we need to make a note of this and watch out for this. We might need it.